modules/xmpp/JingleSessionPC.js

```diff
@@ -1192,6 +1192,13 @@ export default class JingleSessionPC extends JingleSession {
                 return;
             }
 
+            // Skip renegotiation when the selected codec order matches with that of the remote SDP.
+            const currentCodecOrder = this.peerconnection.getConfiguredVideoCodecs();
+
+            if (codecList.every((val, index) => val === currentCodecOrder[index])) {
```

Comment:
Nope, cannot use deepEqual here since `codecList` can be a subset of `currentCodecOrder`. All we need to check is if all the entries from `codecList` exist in `currentCodecOrder` in the same order. There can be extra entries in `currentCodecOrder` but that doesn't necessitate a renegotiation.